### DISCUSSION TITLE
Add on-off switch to humidifier card

### DISCUSSION THREAD

**Discussion by Shulyaka** [home-assistant/frontend#11329] [Dashboard] [answered]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

This is something I am going to do myself when I have some free time, but let's create the issue here since it was requested several times and probably to have a discussion if anyone has something to say on the subject.
It would be more user-friendly if we add the on-off switch to the humidifier card. Currently it is present in the more-info dialog only.

### Describe the behavior you expected

On-off switch for humidifier card without the need to open more-info dialog

### Steps to reproduce the issue

N/A

### What version of Home Assistant Core has the issue?

core-2021.12.7

**Comment by sharoltd** [comment]
Yes, by analogy with the thermostat card...
![Screenshot_3](https://user-images.githubusercontent.com/48355538/149507313-1794050d-189b-4a02-bc51-1fa3d26e95a5.jpg)
![Screenshot_2](https://user-images.githubusercontent.com/48355538/149507317-99104585-5871-42a7-8513-fba13fc9f430.jpg)

**Comment by baradetsky** [comment]
I also need to turn off the hydrostat! According to the power consumption of the smart plug, I can see when the water runs out in it (current_consumption = 0) and I would like to turn off the plug for this condition, but it does not work - the hydrostat (homeassistant.turn_on service) turns it on again! Add the option to disable please!! Or "mode: off" by analogy with "away"

**Reply by Shulyaka** [reply]
Did you try to turn off the hygrostat itself with homeassistant.turn_off?

**Reply by sharoltd** [reply]
> I also need to turn off the hydrostat! According to the power consumption of the smart plug, I can see when the water runs out in it (current_consumption = 0) and I would like to turn off the plug for this condition, but it does not work - the hydrostat (homeassistant.turn_on service) turns it on again! Add the option to disable please!! Or "mode: off" by analogy with "away"

its already works
![Screenshot_7](https://user-images.githubusercontent.com/48355538/151228190-ecd13bb2-3bc1-46d1-88d6-4f7efaee5ef6.jpg)

**Reply by Shulyaka** [reply]
Then it should work. If it does not, please provide the debug logs.

**Comment by Shulyaka** [comment, marked as answer]
Should be fixed in #13443
